README.md

```diff
@@ -1,77 +1,98 @@
 # Powercord Community Guidelines
 
-## 1. Properly structure your plugin's manifest
+These guidelines are set to ensure Powercord's ecosystem stays as awesome as possible, and to avoid malicious plugins,
+or plugins performing badly. The Powercord Staff will enforce **all** of these guidelines and take decisions we consider
+appropriate for plugins not complying with these guidelines, at our sole discretion.
+
+**Note**: We may decide, depending on the context, to let a plugin that would not be compliant with these guidelines
+pass. Those exceptions will be made at our sole discretion.
+
+## The rules
+
+Everything you build for Powercord will have to follow those rules.
+
+### 1. Properly structure your plugin's manifest
 
 All Powercord plugins require a valid `manifest.json` file.
 This file is used internally to give information about the plugin to the end user, as well as define how the plugin
 will behave (Should it be injected in the overlay? Does it have dependencies?)
 
-| Key                  | Description                                                                                                      | Optional |
-|----------------------|------------------------------------------------------------------------------------------------------------------|:--------:|
-| name                 | Display name used in the UI.                                                                                     |          |
-| version              | Version of the plugin. 0.x.x will mean beta and will be flagged as so in the plugins section.                    |          |
-| description          | Short description of what the plugin is doing.                                                                   |          |
-| author               | Name(s) of the author(s).                                                                                        |          |
-| license              | License the plugin is released under.                                                                            |          |
-| permissions          | List of permission the plugin is asking (`keypresses`, `use_eud`, `ext_api`).                                    | x        |
-| dependencies         | List of plugin's dependencies.                                                                                   | x        |
-| optionalDependencies | List of plugin's optional dependencies. Will only affect load order to ensure the plugin is loaded before yours. | x        |
-| appMode              | Where the plugin should be injected. Can be `app`, `overlay`, or `both` (default to `app`).                      | x        |
-
-## 2. Follow Discord and other services ToS
+### 2. Follow Discord and other services ToS
```

Comment:
services' (?)

Reply:
Then it'd also apply do "Discord" right? "Follow Discord's and other services' ToS"

Reply:
that could work